arches/app/media/js/views/components/cards/map.js

```diff
@@ -4,8 +4,7 @@ define([
     'viewmodels/card-component',
     'viewmodels/map-editor',
     'bindings/chosen',
-    'bindings/codemirror',
-    'codemirror/mode/javascript/javascript'
```

Comment:
I think that removing this prevents the syntax highlighting that we use in this interface.  Is that something that you considered, or did you believe that this module was unused?

Reply:
makes sense; I thought that might be the case but wanted to make sure, as I think that was the only reason these were ever included.  thanks!